test/integration/apiserver/cel/admission_test_util.go

```diff
@@ -137,17 +137,17 @@ var (
 		// gvr("admissionregistration.k8s.io", "v1beta1", "validatingwebhookconfigurations"):     true,
 		// gvr("admissionregistration.k8s.io", "v1", "mutatingwebhookconfigurations"):            true,
 		// gvr("admissionregistration.k8s.io", "v1", "validatingwebhookconfigurations"):          true,
-		gvr("admissionregistration.k8s.io", "v1alpha1", "validatingadmissionpolicies"):        true,
-		gvr("admissionregistration.k8s.io", "v1alpha1", "validatingadmissionpolicies/status"): true,
-		gvr("admissionregistration.k8s.io", "v1alpha1", "validatingadmissionpolicybindings"):  true,
-		gvr("admissionregistration.k8s.io", "v1beta1", "validatingadmissionpolicies"):         true,
-		gvr("admissionregistration.k8s.io", "v1beta1", "validatingadmissionpolicies/status"):  true,
-		gvr("admissionregistration.k8s.io", "v1beta1", "validatingadmissionpolicybindings"):   true,
-		gvr("admissionregistration.k8s.io", "v1", "validatingadmissionpolicies"):              true,
-		gvr("admissionregistration.k8s.io", "v1", "validatingadmissionpolicies/status"):       true,
-		gvr("admissionregistration.k8s.io", "v1", "validatingadmissionpolicybindings"):        true,
-		gvr("admissionregistration.k8s.io", "v1alpha1", "mutatingadmissionpolicies"):          true,
-		gvr("admissionregistration.k8s.io", "v1alpha1", "mutatingadmissionpolicybindings"):    true,
+		// gvr("admissionregistration.k8s.io", "v1alpha1", "validatingadmissionpolicies"):        true,
```

Comment:
Does CEL need to test on previous emulated versions? Can we remove these or do we need to keep them to test 1.32/1.31?